geostore/tests/test_views/test_layers.py

```diff
@@ -461,3 +462,81 @@ def test_null_layer_allow_linestring(self):
                                     data={"geom": "LINESTRING(0 0, 1 1)",
                                           "properties": {"toto": "ok"}})
         self.assertEqual(response.status_code, HTTP_201_CREATED, response.json())
+
+
+class LayerPropertyValuesTest(APITestCase):
+    def setUp(self):
+        self.user = UserFactory(
+            permissions=[
+                "geostore.can_manage_layers",
+            ]
+        )
```

Comment:
you should direct use a SuperUserFactory instead of applying permissions. Permissions are already tested in another TesCase

Reply:
Done